shared/common-adapters/markdown/shared.js

```diff
@@ -302,6 +319,30 @@ const rules = {
       return null
     },
   },
+  kbfsPath: {
+    order: SimpleMarkdown.defaultRules.autolink.order - 0.1, // lower than mention
+    match: (source, state, lookBehind) => {
+      const kbfsPathRegex = getKbfsPathRegex(state.markdownMeta)
+      if (!kbfsPathRegex) {
+        return null
+      }
+
+      const matches = SimpleMarkdown.inlineRegex(kbfsPathRegex)(source, state, lookBehind)
```

Comment:
set `const kbfsPathRegex =` above to be `SimpleMarkdown.inlineRegex(createKbfsathRegex())`